utils/hct/pkgesbuild/status-email-template.html

```diff
@@ -11,10 +11,12 @@
 Build Number: $(BUILD_BUILDNUMBER)<br/>
 Reason: $(BUILD_REASON)<br/>
 Source Branch Name: $(BUILD_SOURCEBRANCH)<br/>
-Commit Info: $(BUILD_SOURCEVERSION)<br/>
-<br/>
-Drop Location: \\pkges\Release\Xbox.ShaderCompiler\$(BUILD_DEFINITIONNAME)\$(BUILD_BUILDNUMBER)<br/>
-<br/>
+Commit Info: $(BUILD_SOURCEVERSION)
+</p>
+<p>
+Drop Location: $(XES_DFSDROPDNS)
+</p>
+<p>
 Summary: https://microsoft.visualstudio.com/Xbox/Xbox%20Team/_build/index?buildId=$(BUILD_BUILDID)&_a=summary<br/>
```

Comment:
The github preview is unhappy here because you need to escape the `&` as `&amp;`. Also, <a href="url">text</a> will turn this into a link :)